magik-checks/src/main/resources/nl/ramsolutions/sw/sonar/l10n/magik/rules/NestingLevel.html

```diff
@@ -0,0 +1,34 @@
+<p>The nesting level of nodes should not exceed a defined threshold. Complex code may perform poorly and can be difficult to test thoroughly.</p>
```

Comment:
As with the changelog, I would avoid the term "node" as it is unclear for the "regular developer" (if there is any) without any knowledge of ASTs. Also, add that heavily nested code is less comprehensive/difficult to understand.